### ISSUE TITLE
Have a CONTRIBUTING.md

### ISSUE THREAD

**Issue by FlorianWilhelm** [pyscaffold/pyscaffoldext-markdown#18]
## Describe your use-case
Since PyScaffold itself will have a `CONTRIBUTING.rst` we should make sure that in a new version of this extension, this file is replaced with a corresponding Markdown counterpart. 
  
## Describe the solution you would like to see for your use-case

Maybe there is even a simple rst2md converter, which would allow us to manage only a single template?

**Comment by abravalheri** [comment]
There is this project https://github.com/executablebooks/rst-to-myst from the same folks behind `MyST`, I don't know how simple it is...

**Comment by DrShushen** [comment]
@abravalheri @FlorianWilhelm do you think a new release of the extension can be done at this point? I would quite like to use it in one of my large projects, for example, and the recent MyST based changes etc. would be great to have.

**Comment by abravalheri** [comment]
Hi @DrShushen, thank you very much for the comment.

Although we don't use any `rst2md` converter for the time being, the development version of `pyscaffoldext-markdown` will create a `CONTRIBUTING.md` file without any other plugin.

I did the following test (here I use `-m pyscaffold.cli` as a replacement for `putup` due to the limitations on how [`pip-run`](https://pypi.org/project/pip-run/) works):

```console
$ cd /tmp
$ rm -rf myproj
$ pip-run pyscaffold git+https://github.com/pyscaffold/pyscaffoldext-markdown -- -m pyscaffold.cli $ myproj --markdown

$ tree myproj
myproj
├── AUTHORS.md
├── CHANGELOG.md
├── CONTRIBUTING.md   # <---
├── LICENSE.txt
├── README.md
├── docs
│   ├── Makefile
│   ├── _static
│   ├── authors.md
│   ├── changelog.md
│   ├── conf.py
│   ├── contributing.md
│   ├── index.md
│   ├── license.md
│   ├── readme.md
│   └── requirements.txt
├── pyproject.toml
├── setup.cfg
├── setup.py
├── src
│   └── myproj
│       ├── __init__.py
│       └── skeleton.py
├── tests
│   ├── conftest.py
│   └── test_skeleton.py
└── tox.ini
```

@FlorianWilhelm, do you think we are in a good shape for releasing a new minor version of `pyscaffoldext-markdown`? Is there anything else we should implement?

**Comment by FlorianWilhelm** [comment]
@abravalheri, yes, seems like I forgot this. Let's have a new release to get this out. Do you want to release it? Thanks :-)

**Comment by abravalheri** [comment]
No problems, I updated the CHANGELOG and I created a draft for the new release. I will just wait the CI to pass again.

If we prefer to have the `rst-to-myst` methodology, we can leave this issue open.

**Comment by abravalheri** [comment]
@DrShushen , the new release is out :)

**Comment by DrShushen** [comment]
That's fantastic, thanks!

**Comment by FlorianWilhelm** [comment]
Thanks a lot @abravalheri!
